libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -11,22 +11,38 @@
 
 [role="xpack"]
 [[monitoring-internal-collection]]
+
+
+[IMPORTANT]
+=========================
+{metricbeat} is the recommended method for collecting and shipping monitoring data
+to a monitoring cluster.
+
+If you have previously configured internal collection, migrate to
+using {metricbeat} collection. When using {metricbeat} collection, make
+sure that internal collection is disabled.
+
+You cannot use both {metricbeat} and internal collection.
+
+Learn more about <<monitoring-metricbeat,Monitoring with Metricbeat>>.
```

Comment:
The link ID here is wrong. I think you want monitoring-metricbeat-collection instead of monitoring-metricbeat.